{
  "comments": [
    {
      "key": {
        "uuid": "667f1df8_7518be75",
        "filename": "distroinfo/shell.py",
        "patchSetId": 1
      },
      "lineNbr": 63,
      "author": {
        "id": 200
      },
      "writtenOn": "2019-02-11T16:56:49Z",
      "side": 1,
      "message": "This is failing if I use -f remote and a info_url not finishing in \u0027/\u0027. Could we take this into account, or is it a distroinfo limitation by itself?",
      "revId": "0147fa7297ba17d7ef05000288a6f1e961f52a65",
      "serverId": "6c1dc8ef-8b94-40e4-bd83-c2359d45ecc0",
      "unresolved": false
    }
  ]
}